### PR TITLE
Meowster

### PR DESCRIPTION
**Pull request by HackerstoyanovBG**

### CODE REVIEW

**Comment by HackerstoyanovBG** [comment]
Check the certificates again

File: (B) certificates/certificates.html [hunk 1]
```diff
@@ -11,7 +11,7 @@ <h3 style="margin: 0; background-color: rgb(50, 50, 248);">
             <a href="./certificates" style="margin-left: 8%; font-size: 30px; color: rgb(251, 255, 27);">Certificates</a>
             <a href="./progress" style="margin-left: 1%; font-size: 30px; color: rgb(251, 255, 27);">Progress</a>
             <a href="./School" style="margin-left: 1%; font-size: 30px; color: rgb(251, 255, 27);">School</a>
-            <a href="#navigation" style="margin-left: 1%; font-size: 30px; color: rgb(251, 255, 27);"><span>Games</span></a> <a href="mailto: mitko.vtori@gmail.com" style="color: white; margin-left: 35%; font-size: 50px;">contact</a>
```
Comment (MitkoVtori) [inline (B) certificates/certificates.html:14]: They are alright

File: (B) certificates/certificates.html [hunk 2]
```diff
@@ -59,4 +59,4 @@ <h3>
             </p>
         </h3>
     </body>
-</html>
\ No newline at end of file
+</html>
```
Comment (MitkoVtori) [inline (B) certificates/certificates.html:62]: What have you done here?

Comment (MitkoVtori) [reply (B) certificates/certificates.html:62]: Wait, the free space... okay

**Review by MitkoVtori — APPROVED** [review]
Finally, ready to be published!

**Pull request merged**